frontend/src/__tests__/cypress/cypress/tests/e2e/dashboardNavigation/testManifestLinks.cy.ts

```diff
@@ -0,0 +1,44 @@
+describe('Verify that all the URLs referenced in the Manifest directory are operational', () => {
+  it('Reads the manifest directory, filters out test/sample URLs and validates the remaining URLs', () => {
+    const manifestsDir = '../../../../manifests';
+    cy.log(`Resolved manifests directory: ${manifestsDir}`);
+
+    // Extract URLs from the manifests directory using the registered task
+    cy.task<string[]>('extractHttpsUrls', manifestsDir).then((urls) => {
+      // Filter out Sample/Test URLs
+      const filteredUrls = urls.filter(
+        (url) =>
+          !url.includes('my-project-s2i-python-service') &&
+          !url.includes('clusterip/') &&
+          !url.includes('ClusterIP') &&
+          !url.includes('s2i-python-service') &&
+          !url.includes('user-dev-rhoam-quarkus') &&
+          !url.includes('project-simple') &&
+          !url.includes('example.apps') &&
+          !url.includes('localhost') &&
+          !url.includes('console-openshift-console.apps.test-cluster.example.com/') &&
+          !url.includes('console-openshift-console.apps.test-cluster.example.com') &&
+          !url.includes('repo.anaconda.cloud/repo/t/$'),
+      );
```

Comment:
Maybe It's better to get the URLs (substrings) we want to exclude elsewhere (in the fixtures maybe?) and define a method to filter the urls and just call it in the test.
 Something like 
 ```
const excludedSubstrings: string[] = [
    'my-project-s2i-python-service',
    'clusterip/',
     ...
  ];
  
...

  // Method to determine if a URL should be excluded
  const isUrlExcluded = (url: string): boolean => {
    return excludedSubstrings.some((substring) => url.includes(substring));
  };

....
//Then, in the test:
....
cy.task<string[]>('extractHttpsUrls', manifestsDir).then((urls) => {
      // Filter out Sample/Test URLs
      const filteredUrls = urls.filter((url) => !isUrlExcluded(url));

      // Log filtered URLs for debugging
      filteredUrls.forEach((url) => {
        cy.log(url);
      });
...
```
 This way, we can reuse the method, and also, if at some point we want to add or delete some URL to exclude it's going to be less messy

Reply:
Resolved in latest commit @FedeAlonso , please review thanks.